backend/utils/app.ts

```diff
@@ -0,0 +1,5 @@
+import express from 'express';
```

Comment:
i think this file is not needed since we have a diff app.ts in the backend/ dir already I believe!